### DISCUSSION TITLE
Upcoming AD and CP integration

### DISCUSSION THREAD

**Discussion by Redster999** [Courseplay/Courseplay_FS22#552] [General]
I have wanted to say this for a while now but felt it would be wrong to create an issue just to mention it. I love the fact that the CP team opened up the discussion tab instead.

For me, this is the only FS game that I am playing from the release day. I own all previous games. But ofcourse I am checking the github several times a day. Waiting for the features that CP and AD is hard ;) 

I am missing the automatic load of the seeder/fertilizer while CP is doing the field work, and emptying grass at silo bunkers.
The shovel work that CP made is also really a missed feature. I really hope there is a good dialog between both AD and CP team, they complete this game. I have bought servers online. They keep getting shut down because I am not logging in often enough. So once CP support multiplayer I actually get to use them. At the rate of these guys working, it will come soon. I am not worried.

And when they migrate AD and CP this part it will be even better. Does giants know how many players still waiting for these guys before uninstalling FS19?

About AD

I also had a thought here. Since Stephan dont have a discussion tab, i only post this here anyway. When using AD they already implemented a trigger that when a vehicle is low on gas, go get gas. It works on both gas station and gas silo. Same with the trigger for repairing a vehicle. It only needs to be on an AD-route. It dont need to be saved point. It will find it and the trigger. I play with cp repair disabled because i love the actual drive there with ad. So CP Guys? Please keep this in mind. While on a field. If empty of seeds, find the nearest AD route, refill and back to where CP was working. This is sorely missed. How bout the parking part. Done with field, go find ad-route and park. Hopefully CP could also use a nearby AD route to travel to fieldwork. Because giants driver are still dumber then the annying dog on the farm. 

I am amazed at the work the teams are doing. CP was released on christmas eve and has been updated a lot since it came out. The strategy of completing current features before releasing new features is well thought out. 
I have donated several times to these guys and have probably spent more money on CP then what I bought the game for. These guys don`t bother us for donations. And they have to deal with us nagging them. We ask for things repeatedly without searching the forums. We also sometimes seem to demand them to fix things even though they have told us things takes time.

So thank you for the work. Its well appreciated. Since they are beeing modest, allow me. Please donate to them! They deserve something for all the work being done.

**Comment by Tensuko** [comment]
Give it some time :)
We are working together with AD Team.

About Shovel Mode, that will be different but hopefully much easier, but it also takes some time.

**Reply by Redster999** [reply]
Oh I know. I am waiting. This was just my 2 cents, I understand patience is key here. Glad you brought up discussions again. 

The game is getting better with each new version. So I felt that writing something in issues was wrong since I really like what you guys have done so far. I dont se a lot of error. There is always something weird now and then but nothing that actually gives me a problem big enough for immediate fixing. I would rather have the next feature. Keep up the good work

**Reply by Redster999** [reply]
@Tensuko Can I ask what is AD interface functions?. (ref 7.0.0.20)
My guess is, it doesn`t do much for us yet?

Just wondering ;)

**Reply by mario-fehr** [reply]
> @Tensuko Can I ask what is AD interface functions?. (ref 7.0.0.20)
> My guess is, it doesn`t do much for us yet?
> 
> Just wondering ;)

AD developer can use them too implement communication with CP

**Reply by Redster999** [reply]
oooh. Thanks. So it will do much for us later 👍

**Comment by ProNeli** [comment]
Wir den das Silo verdichten auch wieder Kommen ? Das vermisse ich am meisten das man auch zu 2. mal wieder häckseln kann mit ad und cp ?

**Reply by Tensuko** [reply]
Ja
